realm/src/main/java/io/realm/BaseRealm.java

```diff
@@ -137,6 +137,17 @@ public boolean isAutoRefresh() {
     }
 
     /**
+     * Indicates if the Realm is currently in a write transaction.
+     * @return the write transaction status.
```

Comment:
The below would be more in line with what we write elsewhere:

`@return {@code true} if inside a write transaction, {@code false} otherwise.`